brainiak/funcalign/srm.py

```diff
@@ -11,9 +11,9 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
-"""Probabilistic Shared Response Model (SRM)
+"""Shared Response Model (SRM)
 
-This implementation is based on the following publications:
+The implementations are based on the following publications:
```

Comment:
Could you please change the second reference to our Arxiv paper?
https://arxiv.org/abs/1608.04647